wiki/FAQ/en.md

```diff
@@ -274,71 +358,126 @@ _Detailed explanation of scoring can be found at the game style's own respective
+
+#### Does osu! has keyboard shortcuts?
+_Full list: [Shortcut Key Reference][Shortcut Key Reference wikilink]_
+
+Pressing `Insert` key (in default keyboard setting) will hide, pause (if playing) and mute osu! into the Windows sidebar. Find the osu!logo in the Windows sidebar, and click on it to return osu! to the state before pressing the key. It is called the _Boss key_, and can be modified in _Change keyboard bindings_ (in Options, under Keyboard section), located at the end of the Universal section.
+
+Another interesting shortcut is `Alt`+`Enter`, which makes osu! goes fullscreen/windowed mode. It does not work during gameplay, so please use it anywhere outside gameplay.
+
+#### What should I backup when fresh installing osu! on a new device?
+The whole `osu!` folder, including hidden files, is the short answer. On the new device after running `osu!install.exe`, copy the backup `osu!` folder and merge to the new device's `osu!` folder. If it ask for merging `.dll` or `.exe` files, ignore it. Allow merging for everything else, including `.cfg` and `.db` files.
+
+### osu!
 #### Why am I seeing random white lines on screen every time I play?
-Make sure that the smoke/skywriting key binding is not the same as the keybinding you use at playing.
+Make sure that the smoke/skywriting key binding is not the same as the key binding used to tap with when playing.
+
+![osu! smoke effect][osu_smoke image]
+
+_Usage of Smoke_
+
+![osu! smoke settings][osu_smoke_set image]
+
+_Smoke in key bindings settings; hold `C` key to trigger by default in osu!_
+
+#### Why do I not have the max possible combo when I have no misses?
+![Image showing a slider with sliderticks, with top part during Edit and bottom part during play][Sliderticks image]
 
-Image | Description
-:---: | :---
-![osu! smoke effect][osu_smoke image] | Usage of "Smoke"
-![osu! smoke settings][osu_smoke_set image] | "Smoke" in key bindings settings
+_Top image shows a slider with all the sliderticks shown in Edit mode; Bottom image shows the same slider during play, with some sliderticks obstructed by start and end circle of the slider._
 
-### Why do I not have the max possible combo when I have no misses?
-You released a slider too early or you barely clicked the hit-circle. [Answer's Source][reddit osugame wiki]
+**You released a slider early (meaning that you released the ball before it had finished the ending slider point).**
+
+A slider is composed of a starting circle point, spaced smaller circles known as sliderticks, repeat points (if used, it will be treated as starting point instead; it is a proper slider in reverse), and an ending circle point. Each of those points on a slider will add one to your combo. Note that the starting circle point is worth 30 points, the sliderticks are worth 10 points each, the repeat points are worth 30 points each, and the end circle point is worth 300 (or geki), 100 (or katu), or 50 points depending on how the slider was completed.
```

Comment:
When I edited your draft on Discord, I forgot what reverse arrows were, so I called them "repeat points" \:p
If you want, I think you should replace `repeat points` with `reverse points` or something like so